blog/docs/articles/demystifying-activator-on-path.md

```diff
@@ -0,0 +1,182 @@
+# Demystifying Activator on path
+
+**Author: [Stavros Kontopoulos](https://twitter.com/s_kontopoulos), Principal Software Engineer @ RedHat**
+
+**Date: 2023-10-11**
+
+_In this blog post you will learn how to recognize when activator is on the data path and what it triggers that behavior._
+
+A knative service can operate in two modes:proxy mode and serve mode.
+When in proxy mode Activator is on the data path and it will stay on path until certain conditions are met.
```

Comment:
```suggestion
When in proxy mode, Activator is on the data path, and it will stay on the path until certain conditions (more on this later) are met.
```